R/arg_checks.R

```diff
@@ -291,9 +291,35 @@ check_model_names <- function(model_names) {
     stop(base::paste("Invalid model selected! List of models allowed:",
                      base::paste(available_models, collapse = ", ")))
   }
+  if ("lstm_keras" %in% model_names_unlist) {
+    model_names_unlist <- check_tensorflow(model_names)
+  }
   return(model_names_unlist)
 }
 
+#' Check if tensorflow is properly installed
+#' @description
+#' This function ensures that the user has Python and Tensorflow (version <= 1.14) installed.
+#' If these criteria are violated, then the user cannot use the 'lstm_keras' model.
+#' @param model_names A list or vector of strings
+#' @return model_names: vector of strings
```

Comment:
The comment about the `model_names` both `@param` and `@return` only specify the types.
Even more helpful would be the meaning in what it contains, ie.
```
@param  model_names A list or vector of strings representing the model names to be used
@return model_names: vector of strings the input model_names without 'lstm_keras' depending on that algorithms prerequisites.
```